wgsl/index.bs

```diff
@@ -845,7 +849,7 @@ That is, a declaration at module scope may be referenced by source text
 that follows *or precedes* that declaration.
 
 It is a [=shader-creation error=] if any module scope declaration is recursive.
-That is, there must be no cycles among the declarations:
+That is, there can be no cycles among the declarations:
```

Comment:
I think I still prefer "must" here...

Hm.  It feels like there's a singular/plural confusion here even in the old text.

How about rewording:

> That is, no cycles can exist among the declarations: